doc/src/jbdiagnose.md

```diff
@@ -45,7 +45,19 @@ plotjbdiag -l 50 -t vercor -p QQ -r ./ -e IRELAND75L65
 The development and implementation of the HARMONIE data assimilation are now in a quite advanced stage with pre-operational testing at most of the HIRLAMmember weather services. At the same time, the technical and scientific knowledge about various components of this data assimilation is not so widely spread and a joint effort is needed forthe validation of these locally installed system. One of the important components of the HARMONIE data assimilation is the background error statistics. A tool for diagnostics of the background error statistics directly from the background error statistics files has therefore been developed and applied to the recently developed statistics files for HARMONIE at mesoscale (AROME)resolution (2.5 km grid resolution) at SMHI, met.no, FMI, DMI, KNMI and AEMET. This note describes the diagnostic tool (jbdiagnose) and presents results from a comparison of the different statistics from the AROME implementations. Since this diagnostic software is quite fresh, there may certainly be coding errors that could affect the results. Any comments on the comparison and the graphs presented here are therefore most welcome.
 
 ## 2 The diagnostic tool
-The first part of the diagnostic tool is a simple standalone fortran program (jbdiagnose.F90) that reads the two main background error statistics files (*.bal and *.cv), calculates various diagnostic quantities and writes these diagnostic quantitiesin simple ASCII files that can be used by, for example, GNUPLOT to produce various graphs. The input background error statistics files are opened with the local file names stabal96.bal and stabal96.cv and a namelist variable gsizein, describing the grid resolution, and namelist variables describing the vertical levels (AHALF, BHALF) are needed. Various information about the domain geometry is extracted from the input files. Oneexample of a script to run the diagnostic tool is provided in jbdiagnose.sh. The output ASCII files are stored in the
+The first part of the diagnostic tool is a simple standalone fortran program (jbdiagnose.F90) that reads the two main background error statistics files (jb\_filename.bal and jb\_filename.cv), calculates various diagnostic quantities and writes these diagnostic quantitiesin simple ASCII files that are then used by _plotjbdiag_ to produce various graphs. The input background error statistics files are opened with the local file names _stabal96.bal_ and _stabal96.cv_ and a namelist variable _gsizein_, describing the grid resolution, and namelist variables describing the vertical levels (_ahalf\_nl_ and _bhalf\_nl_, ) are needed.
+
+The following vertical level definitions are available for use:
+| Name       | Description                      |
+| ---------- | -------------------------------- |
+|hirlL60.def |HIRLAM 60 level definition        | 
```

Comment:
```suggestion
The following vertical level definitions are available for use:

| Name       | Description                      |
| :--      | :--                                 |
|hirlL60.def |HIRLAM 60 level definition        | 
```

Reply:
The empty line before the markdown table is need or else you wil get: 

![Screenshot from 2024-06-28 15-06-04](https://github.com/ACCORD-NWP/AccordDaTools/assets/41205070/019994cf-611f-485c-b56a-0005d44283d1)